config/mongo.js

```diff
@@ -1,7 +1,11 @@
 import mongoose from "mongoose";
 
-const connectMongo = async () => mongoose.connect(process.env.MONGODB_URI).then((data) => console.log("Hello connected")).catch((res) => console.log("Here is error"));
-
-console.log("Connected or not",connectMongo)
+const connectMongo = async () => {
+    try{
+        await mongoose.connect(process.env.MONGODB_URI).then(() => console.error("DB connected")).catch((err) => console.error("DB not connected",err));
+    }catch(err){
```

Comment:
- Because it is wrap in. `try/catch` the `catch` is not needed
- now you are using `await` the `then` is not needed

Something like this is easier to read
```js
await mongoose.connect(process.env.MONGODB_URI)
console.log("DB connected"))
```